genesyscloud/responsemanagement_response/resource_genesyscloud_responsemanagement_response_schema.go

```diff
@@ -143,7 +143,7 @@ func ResourceResponsemanagementResponse() *schema.Resource {
 				Description: `One or more libraries response is associated with. Changing the library IDs will result in the resource being recreated`,
 				Required:    true,
 				ForceNew:    true,
-				Type:        schema.TypeList,
+				Type:        schema.TypeSet,
```

Comment:
I think this requires a state migration, otherwise Terraform will force recreate any of this resource type on the next apply because the data is stored differently on the state file.

Reply:
I deployed a response with the latest published version and then upgraded to the locally compiled and saw no issues (`No changes.`)

I don't think this is a problem because in the state file `library_ids` is just a JSON string array with no mention to TypeList or Set